packages/react-reconciler/src/ReactFiber.new.js

```diff
@@ -29,7 +29,7 @@ import {
   enableScopeAPI,
   enableBlocksAPI,
 } from 'shared/ReactFeatureFlags';
-import {NoFlags, Placement, StaticMask} from './ReactFiberFlags';
+import {NoFlags, Placement, StaticMask} from 'shared/ReactFiberFlags';
```

Comment:
This is not a type change.

Reply:
it's a type+const-only, but you're right -- ReactWorkTags was similar and I moved that back based on your feedback. will do this.